frontend/src/pages/detail/components/Carousel/Carousel.styled.ts

```diff
@@ -23,28 +25,29 @@ const CarouselImage = styled.img`
 
 export const Image = styled(CarouselImage)<{
   position: "current" | "next" | "prev" | "hidden";
+  length: number;
   noTransition?: boolean;
 }>`
   position: absolute;
   transition: ${({ noTransition }) =>
     noTransition ? "none" : "all 0.5s ease-in-out"};
 
-  ${({ position }) => {
+  ${({ position, length }) => {
     switch (position) {
       case "current":
         return `
-        width: 45rem;
-        height: 25rem;
+        width: 43.75rem;
+        height: 23.75rem;
         opacity: 1;
-        transform: translateX(-15rem);
+        transform: translateX(${length > 1 ? "-15rem" : "0"});
```

Comment:
```suggestion
        transform: translateX(${isCentered ? "-15rem" : "0"});
```
`length` prop을 받아서 styled 내부에서 비교 연산을 하기 보다는 컴포넌트에서 `length > 1` 와 같은 비교 연산을 한 결과를 넘겨주는 것은 어떨까요??

Reply:
좋습니다 ㅎㅎ 수정했습니다 ~ !

[refactor: 캐러셀 isSingleImage 계산 외부로 이동](https://github.com/woowacourse-teams/2025-moaon/pull/146/commits/9207270dfcc9821aa992192c7e73fcfcac1665da)